src/resources/Pipelines/StatementGroups/StatementGroupsInterfaces.ts

```diff
@@ -7,17 +7,47 @@ import {
 
 export interface StatementGroupList {
     groups: StatementGroupModel[];
+
+    /**
+     * The total number of matching statements across all pages of results.
+     */
     totalCount: number;
+
     groupComposition: PipelineGroupsComposition;
 }
 
 export type StatementGroupModel = PermanentStatementGroup | CampaignStatementGroup;
 
 interface StatementGroupModelBase {
+    /**
+     * The unique identifier of the statement group.
+     */
     id: string;
+
+    /**
+     * The name of the statement group.
+     */
     name: string;
+
+    /**
+     * The intented purpose of this statement group.
+     */
     description?: string;
+
+    /**
+     * The unique identifier of the condition that must be satisfied for a request to be processed by the statement group.
+     * A query cannot be routed through a query pipeline that does not have a condition unless:
+     * - That query pipeline is set as the default query pipeline.
+     * - That query pipeline is enforced through the pipeline parameter of the query itself, in which case the query pipeline condition is bypassed.
```

Comment:
You can keep only the first line. Those lines describe the QueryPipeline routing 
```suggestion
```